app/lib/cfe_connection.rb

```diff
@@ -0,0 +1,60 @@
+class CfeConnection
+  CFE_HOST = "https://check-financial-eligibility-staging.cloud-platform.service.justice.gov.uk/".freeze
```

Comment:
When we eventually want to point our production at production CFE is there a smart system for service discoverability or is it just a case of different URLs in env vars?